codesy/base/stripe_events.py

```diff
@@ -0,0 +1,104 @@
+import json
+from decimal import Decimal
+
+from django.conf import settings
+from django.db import models
+
+from six import with_metaclass
+
+
+class StripeAccount(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL)
+    account_id = models.CharField(max_length=100, blank=True)
+    secret_key = models.CharField(max_length=100, blank=True)
+    public_key = models.CharField(max_length=100, blank=True)
+    available_balance = models.DecimalField(
+        max_digits=6, decimal_places=2, blank=True, default=0)
+    verification = models.TextField(default='', blank=True)
+
+    def identity_verified(self):
+        verification = json.loads(self.verification)
+        return not verification.due_by
+
+
+class StripeEvent(models.Model):
+    event_id = models.CharField(primary_key=True, max_length=100, blank=True)
+    type = models.CharField(max_length=100, blank=True)
+    message_text = models.TextField()
+    processed = models.BooleanField(default=False)
+
+    def message(self):
+        return json.loads(self.message_text)
+
+    def save(self, *args, **kwargs):
+        web_hooks[self.type].process(self)
```

Comment:
Looks up the webhook associated with the event type (e.g. 'account.updated') and runs it's process.